tests/tt_metal/tt_metal/unit_tests_common/common/test_dispatch.cpp

```diff
@@ -87,6 +87,107 @@ static void test_sems_across_core_types(CommonFixture *fixture,
     }
 }
 
+TEST_F(CommonFixture, TestTensixBlank) {
```

Comment:
This might be a duplicate, we have `DeviceLoadBlankKernels` test to load blank kernels on all tensix cores

Reply:
that test doesn't run w/ either unit_tests_fast_dispatch or unit_tests, do you know which suite it runs with?

Reply:
it is in `unit_tests_common/basic/test_device_init.cpp` so I would assume that it should be picked up by both suites